.circleci/config.yml

```diff
@@ -439,6 +470,22 @@ jobs:
                     tests/pytest_tests/unit_tests/test_launch/ \
                     tests/pytest_tests/unit_tests/test_reports/
 
+      - when:
+          condition: { equal: [<<parameters.tests>>, importer] }
+          steps:
+            - run:
+                name: Run tests not owned by SDK
+                no_output_timeout: 10m
+                command: |
+                  CI_PYTEST_SPLIT_ARGS=" \
+                    --splits $CIRCLE_NODE_TOTAL \
+                    --group $(( $CIRCLE_NODE_INDEX + 1 ))" \
+                  tox run -v \
+                    -e <<#parameters.with_core>>core-<</parameters.with_core
```

Comment:
do you need all the extra installs for unit-tests as well? if not you could still include it in the `other` shard

Reply:
alternatively you could install them with the rest of the dependencies?